### DISCUSSION TITLE
Plugin installation error

### DISCUSSION THREAD

**Discussion by JMing96** [ueberdosis/tiptap#5197] [General Questions] [answered]
I'm using tiptap on macOS because I need to get https://registry.npmjs.org/@tiptap-pro%2fextension-invisible-characters - Not found when installing other dependencies
Error, but I don't have this dependency installed, I tried to find and delete this dependency and didn't find it, how can I remove or fix this
<img width="981" alt="image" src="https://github.com/ueberdosis/tiptap/assets/31053017/d5cb5936-4dbd-44a8-b630-ee7a8660f8dd">

**Comment by nperez0111** [comment]
To access Tiptap Pro extensions, you need to pull from our tiptap registry, not from NPM's registry.

To do this, see the setup guide on our dashboard page: https://cloud.tiptap.dev/pro-extensions

It will tell you how to configure NPM to pull from our registry using your unique auth token

**Reply by JMing96** [reply]
Now I don't need a p ro plugin with this error, how can I fix this error, I tried to search the local file but didn't find the extension-invisible-characters plugin, please how can I do it

**Reply by nperez0111** [reply, marked as answer]
Nothing should depend on the extension-invisible-characters (in our repos). So this means that your package.json is specifying that dependency. If you do not need it then you can remove it from your `"dependencies"`

This isn't any issue on our side. Whatever you are trying to install is using that package.

If you are trying to run our templates, then they depend on multiple pro extensions and you would need to do as I said earlier

**Reply by JMing96** [reply]
Okay, thanks. I'll check again
